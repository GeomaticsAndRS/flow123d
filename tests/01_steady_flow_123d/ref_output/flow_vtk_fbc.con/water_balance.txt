	"time"	"region"	"quantity"	"flux"	"flux_in"	"flux_out"	"mass"	"source"	"source_in"	"source_out"	"integrated_flux"	"integrated_source"	"error"
	0	"1d_channel"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"2d_fracture_1"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"2d_fracture_2"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"3d_cube"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"IMPLICIT BOUNDARY"	"water_volume"	-2.07188e-12	-2.21069	2.21069	0	0	0	0	0	0	0
