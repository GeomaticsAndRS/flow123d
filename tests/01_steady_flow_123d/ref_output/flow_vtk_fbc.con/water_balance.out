# ---------------------------------------- WATER BALANCE ----------------------------------------
# Time of computed water balance: inf


# Boundary water balance:
# [boundary_id] [label]                  [total_balance][total_outflow][total_inflow][time]        
# -----------------------------------------------------------------------------------------------
  -3            IMPLICIT BOUNDARY          1.98664e-10   2.21069       -2.21069      inf           
# -----------------------------------------------------------------------------------------------
# total boundary balance                   1.98664e-10   2.21069       -2.21069      


# Source fluxes over material subdomains:
# [region_id]   [label]                  [total_balance]                            [time]        
# -----------------------------------------------------------------------------------------------
  1             1d_channel                 0                                         inf           
  2             2d_fraction_1              0                                         inf           
  3             2d_fraction_2              0                                         inf           
  4             3d_cube                    0                                         inf           
# -----------------------------------------------------------------------------------------------
# total sources balance                    0             


